nntrainer/layers/fc_layer.h

```diff
@@ -26,20 +26,17 @@ namespace nntrainer {
  * @class   FullyConnecedLayer
  * @brief   fully connected layer
  */
-class FullyConnectedLayer : public LayerV1 {
+class FullyConnectedLayer : public LayerImpl {
 public:
   /**
    * @brief     Constructor of Fully Connected Layer
    */
-  template <typename... Args>
-  FullyConnectedLayer(unsigned int unit_ = 0, Args... args) :
-    LayerV1(args...),
-    fc_props(props::Unit(unit_)) {}
+  FullyConnectedLayer() : LayerImpl(), fc_props(props::Unit(0)) {}
```

Comment:
slightly off the topic, is Unit of zero valid prop? else I think we can leave it as empty.

Reply:
will update to empty.